### PR TITLE
Change zeit to vercel, add vercel config files

### PR DESCRIPTION
**Pull request by bibixx**
## Motivation
Recently Zeit has been renamed to Vercel (https://vercel.com/blog/zeit-is-now-vercel). Because of that they have introduced new filenames – `vercel.json` (previously `now.json`) and `.vercelignore` (previously `.nowignore`).
## This PR introduces
* Renaming Zeit to Vercel
* Adding new file types to use the same icon as were used for files for now

### CODE REVIEW

File: config.cson [hunk 1]
```diff
@@ -6780,11 +6780,13 @@ fileIcons:
...
 
-	Zeit:
-		icon: "zeit"
+	Vercel:
+		icon: "vercel"
 		match: [
 			[/^now\.json$/i, "auto-grey", priority: 2, uses: "JSON"]
 			[".nowignore", "auto-maroon", uses: "Ignore List"]
+			[/^vercel\.json$/i, "auto-grey", priority: 2, uses: "JSON"]
+			[".vercelignore", "auto-maroon", uses: "Ignore List"]
```
Comment (Alhadis) [inline config.cson:6789]: Might be better to merge these into the existing entries:

~~~cson
match: [
	[/^(vercel|now)\.json$/i, "auto-grey", priority: 2, uses: "JSON"]
	[/^\.(vercel|now)ignore$/i, "auto-maroon", uses: "Ignore List"]
]
~~~

File: styles/icons.less [hunk 2]
```diff
@@ -924,7 +924,7 @@
 .yorick-icon:before        { .fi; content: "\1f480";top: 2px; }
 .yui-icon:before           { .fi; content: "\ea00"; top: 2px; }
 .zbrush-icon:before        { .fi; content: "\e9f2"; top: 2px; font-size: 16px; }
-.zeit-icon:before          { .fi; content: "\25B2"; top: 2px; }
+.vercel-icon:before        { .fi; content: "\25B2"; top: 2px; }
```
[the remark this answers is not in the document]
Comment (bibixx) [reply styles/icons.less]: Thanks for quick response! Totally makes sense, fixed it 🙂

**Comment by Alhadis** [comment]
Perfect! Thank you! 👍

**Comment by bibixx** [comment]
Hey @Alhadis is there any possibility that this change will be pushed to the VSCode package in the near future?

**Comment by Alhadis** [comment]
Hey @bibixx, sorry. I haven't had time to publish *any* release recently, which I hope to rectify tonight (seeing as my ability to get online is somewhat… sporadic lately).

Don't worry, I always publish an update to both Atom and VSCode's package registries at the same time (so they don't fall out of sync).

**Comment by Alhadis** [comment]
Also, posting this purely because I have nowhere else to share it:

figure-1

You know GitHub's running out of recs when it starts recommending your own projects…

**Comment by Alhadis** [comment]
Sorry for the wait, I had to finish the obligatory pre-release icon scrounging ritual. 😉 

You should see v1.0.25 in VSCode's extension updates panel. 👍

**Pull request merged**